src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -198,7 +210,17 @@ private static boolean isTosca(Map<?,?> obj) {
                 String templateId = templateE.getKey();
                 NodeTemplate template = templateE.getValue();
 
-                EntitySpec<VanillaSoftwareProcess> thisNode = new ToscaComputeToVanillaConverter(mgmt).toSpec(templateId, template);
+                EntitySpec<? extends Entity> thisNode = null;
+                try {
+                    thisNode = EntitySpec.create((Class<Entity>) Class.forName(template.getType()));
```

Comment:
what is the use case for this block?  feels wrong to accept java classes like this.  instead we should compare against catalog items first and then attempt a java lookup.  (which is how the existing catalog lookup does it).  needs some thinking to handle this cleanly, but looking up the java type doesn't seem good.  also won't this code ignore any properties (aka brooklyn config) and other settings on the template?